test/frontmatter-linter-test.js

```diff
@@ -155,6 +155,7 @@ describe('FrontmatterLinter', function () {
       'prs.accepted must be the URL for the original pull request on emberjs/rfcs, for example: https://github.com/emberjs/rfcs/pull/123',
       'start-date must be a date formatted YYYY-MM-DD',
       'teams must be a list of one or more Ember teams',
+      'release-date must be a date formatted YYYY-MM-DD',
```

Comment:
Is this correct though? On an early stage, do we want submitters of RFCs to see the message 'release-date must be a date formatted YYYY-MM-DD'? Because that is what this test is saying.

Reply:
Also noticing, pretty sure the messages are incorrect if we're now requiring iso dates.